NBXplorer.Client/JsonConverters/CachedSerializer.cs

```diff
@@ -52,12 +52,13 @@ internal bool CanConvert(Type objectType)
 				return Converter.CanConvert(objectType);
 			}
 		}
-		public CachedSerializer(Network network)
+
+		public CachedSerializer(NBXplorerNetwork network)
 		{
 			if (network == null)
 				throw new ArgumentNullException(nameof(network));
-			cachedConverter.Add(new CachedConverter(new BitcoinStringJsonConverter(network)));
-			cachedConverter.Add(new CachedConverter(new DerivationStrategyJsonConverter(new DerivationStrategy.DerivationStrategyFactory(network))));
+			cachedConverter.Add(new CachedConverter(new BitcoinStringJsonConverter(network?.NBitcoinNetwork)));
```

Comment:
NBitcoin does not support null network here, so should be `network.NBitcoinNetwork`